test/cds.ql.test.js

```diff
@@ -287,7 +286,7 @@ describe('cds.ql → cqn', () => {
                 'in',
                 { val: args },
```

Comment:
That seems completely wrong, should be something reflecting `args in (...)` while this one here says it's `args in 'args'`, which isn't even valid SQL
Fix to be same as by compiler.